src/parse-stream.js

```diff
@@ -103,6 +104,11 @@ export default class ParseStream extends Stream {
       return;
     }
 
+    // map tag
+    for (let i = 0; i < this.tagMappers.length; i++) {
+      line = this.tagMappers[i].call(this, line);
```

Comment:
what happens if you have multiple tagMappers for the same tag?

Reply:
It'll execute the first matching mapper. I'll add a test for that

Reply:
should also have a note in the docs